partner_identification_gln/tests/test_gln.py

```diff
@@ -6,19 +6,20 @@
 
 
 class TestGLN(SavepointCase):
-    def setUp(self):
-        super(TestGLN, self).setUp()
-        self.partner = self.env["res.partner"].create({"name": "TestGLN"})
-        self.partner2 = self.env["res.partner"].create({"name": "TestGLN2"})
-        pc = self.env.ref(
-            "partner_identification_gln." "partner_identification_gln_number_category"
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
```

Comment:
`cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))`
Maybe can be added as well ?

Reply:
good tip :wink: